src/+replab/+str/Table.m

```diff
@@ -0,0 +1,365 @@
+classdef Table < replab.Str
+% Helper to display tables on the REPL
+%
+% >>> replab.str.Table(cell(4,4), 2:3, 'llcc') % unreadable
```

Comment:
Correct the example
```
% Example:
%    >>> commandToTry
%           expectedOutput

Reply:
If you can't really check the `expectedOutput` because of variations on the printing, just make the line silent by adding a `;`. It still shows the syntax.